### PR TITLE
fix: added .vscode and .nvmrc in .gitignore

### PR DESCRIPTION
**Pull request by adrijshikhar**
#### Short description of what this resolves:
added .vscode in .gitignore

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes

### CODE REVIEW

**Comment by kushthedude** [comment]
From next time please avoid micro-PRs. Bundle micro-PRs into a bigger one. We want to ensure that devs who review PRs use their time efficiently.

File: .gitignore [hunk 1]
```diff
@@ -21,7 +21,7 @@ testem.log
 /kubernetes/images/frontend/node_Modules/
 .env
 .idea
-
```
Comment (kushthedude) [inline .gitignore]: Don't remove this line.

**Comment by snitin315** [comment]
Follow Semantic PR naming.

**Comment by iamareebjamal** [comment]
Add both entries in a single PR

File: .gitignore (hunk 1 above)
Comment (adrijshikhar) [reply .gitignore]: i did not remove it, it was my vscode I guess. i will take care of that.

File: .gitignore [hunk 2]
```diff
@@ -21,7 +21,7 @@ testem.log
...
+.vscode
```
Comment (iamareebjamal) [inline .gitignore]:
```suggestion
.vscode

```

**Pull request merged**